### DISCUSSION TITLE
It´s possible to generate Heatmaps with supervision ?

### DISCUSSION THREAD

**Discussion by Rasantis** [roboflow/supervision#601] [Q&A] [answered]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

It´s possible to generate Heatmaps with supervision ? if not , it would be an a excellent feature to the next version ! use supervision, and yolov8 to generate heatmaps based on tracking people...

**Comment by SkalskiP** [comment]
Hi @Rasantis! 👋🏻 Thanks a lot for your interest in supervision. First, let me convert this issue into a discussion and move it to the Q&A section.

**Comment by SkalskiP** [comment, marked as answer]
@Rasantis is this what you are looking for? https://supervision.roboflow.com/annotators/#supervision.annotators.core.HeatMapAnnotator

**Reply by Rasantis** [reply]
yeah @SkalskiP exactly, thanks !

**Reply by SkalskiP** [reply]
My pleasure!
